example/node/index.js

```diff
@@ -27,30 +28,12 @@ async function setAuthTokenWithOAuthCredentials() {
     return;
   }
 
-  let authToken;
   const clientId = process.env.CLIENT_ID;
   const clientSecret = process.env.CLIENT_SECRET;
   printOut('Requesting auth token with client id from env vars:', clientId);
 
-  await axios({
```

Comment:
Note that these two lines are no longer needed:
```
const dotenv = require('dotenv').config({ path: '../../.env' });
const axios = require('axios');
```

Similary, import for `axios` in storybook examples is not needed anymore.

Reply:
`const dotenv = require('dotenv').config({ path: '../../.env' });` is still needed unfortunately. I moved it above import of sentinelhub-js things.

Removed `const axios = require('axios');` where it was applicable.

Reply:
True, I was mislead by the editor that marked variable `dotenv` as unused.

To make it clearee, I would suggest changing the line to just:
```
require('dotenv').config({ path: '../../.env' });
```

But that's already nitpicking, feel free to resolve either way.

Reply:
It looks better without `const dotenv = ` anyway so I changed it.